.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request, workflow_call, workflow_dispatch]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set CACHE_DATE
+      run: eval echo 'CACHE_DATE="$(date --utc +%Y%m%dT%H%M%SZ)"' >> $GITHUB_ENV
```

Comment:
Why the indirection of `eval` here?

Reply:
Without `eval`, `$(date --utc +%Y%m%dT%H%M%SZ)` is just part of the string. See https://github.com/victorlin/github-test/runs/6277772107

Reply:
Sorry, yes, I understand that. But why not just?

```
echo "CACHE_DATE=$(date --utc +%Y%m%dT%H%M%SZ)" >> $GITHUB_ENV
```

Reply:
Oh good call, I forgot how `'` vs `"` work here. Done.